pd/src/genesis.rs

```diff
@@ -47,29 +48,19 @@ impl Allocation {
     }
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ValidatorPower {
     pub validator: Validator,
     pub power: tendermint::vote::Power,
 }
```

Comment:
Non-blocking on this PR, but we should add docs to this struct that explains its purpose, relative to `Validator` and `ValidatorStatus`.  The `Validator` domain type has all of the config data for a validator, and the `ValidatorStatus` has the voting power, (in the future) the validator state (inactive/active/slashed/unbonding), etc.

As we expand the `ValidatorStatus` structure, we'll need to replicate its fields here, or change this structure to be a `Validator`, `ValidatorStatus` pair.

The only reason not to change the structure to be a pair would be that the `ValidatorStatus` references a validator by identity key, so if we had the validators specified that way in the genesis config, it would be necessary to copy the identity key in two different places. Maybe that's better than maintaining a custom, genesis-only type?

Reply:
Another option: perhaps we shouldn't specify a power at all, and instead compute the power from the genesis allocations, which can include delegation tokens?